From f4c1f1ceab9dba5335832248e51fab3f15a5b4c9 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Thu, 11 Jun 2020 00:50:18 -0700
Subject: [PATCH 1171/1239] Revert "drivers: mmc: Enable for OcteonTX3
 platforms"

This reverts commit bcb0cad950d66199a4361de4a7e7111dff0ac6d6.

Reason for revert: Incorrect platform name

Change-Id: Idd656aae93891d34aae7f39bc186ca3c40050eb9
Signed-off-by: Suneel Garapati <sgarapati@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/29830
Reviewed-by: Suneel Garapati <sgarapati@caviumnetworks.com>
Tested-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 drivers/mmc/Kconfig                | 2 +-
 drivers/mmc/octeontx_hsmmc.c       | 7 -------
 include/configs/octeontx3_common.h | 2 +-
 3 files changed, 2 insertions(+), 9 deletions(-)

diff --git a/drivers/mmc/Kconfig b/drivers/mmc/Kconfig
index 888a78241a..19da6f88e2 100644
--- a/drivers/mmc/Kconfig
+++ b/drivers/mmc/Kconfig
@@ -291,7 +291,7 @@ config MMC_PCI
 
 config MMC_OCTEONTX
 	bool "Marvell OcteonTX Multimedia Card Interface support"
-	depends on (ARCH_OCTEONTX || ARCH_OCTEONTX2 || ARCH_OCTEONTX3)
+	depends on (ARCH_OCTEONTX || ARCH_OCTEONTX2)
 	depends on DM_MMC
 	help
 	  This selects the OcteonTX Multimedia card Interface.
diff --git a/drivers/mmc/octeontx_hsmmc.c b/drivers/mmc/octeontx_hsmmc.c
index a3055c4664..95544b1960 100644
--- a/drivers/mmc/octeontx_hsmmc.c
+++ b/drivers/mmc/octeontx_hsmmc.c
@@ -39,17 +39,10 @@
 #define MMC_DEFAULT_DATA_OUT_TAP		39
 #define MMC_DEFAULT_HS200_CMD_IN_TAP		24
 #define MMC_DEFAULT_HS200_DATA_IN_TAP		24
-#if CONFIG_IS_ENABLED(ARCH_OCTEONTX2)
 #define MMC_DEFAULT_HS200_CMD_OUT_TAP	(otx_is_soc(CN95XX) ? 10 : 5)
 #define MMC_DEFAULT_HS200_DATA_OUT_TAP	(otx_is_soc(CN95XX) ? 10 : 5)
 #define MMC_DEFAULT_HS400_CMD_OUT_TAP	(otx_is_soc(CN95XX) ? 10 : 5)
 #define MMC_DEFAULT_HS400_DATA_OUT_TAP	(otx_is_soc(CN95XX) ? 5 : 3)
-#else
-#define MMC_DEFAULT_HS200_CMD_OUT_TAP		5
-#define MMC_DEFAULT_HS200_DATA_OUT_TAP		5
-#define MMC_DEFAULT_HS400_CMD_OUT_TAP		5
-#define MMC_DEFAULT_HS400_DATA_OUT_TAP		3
-#endif
 #define MMC_DEFAULT_HS200_CMD_OUT_DLY		800	/* Delay in ps */
 #define MMC_DEFAULT_HS200_DATA_OUT_DLY		800	/* Delay in ps */
 #define MMC_DEFAULT_HS400_CMD_OUT_DLY		800	/* Delay in ps */
diff --git a/include/configs/octeontx3_common.h b/include/configs/octeontx3_common.h
index ccfa12b22b..4af40bd740 100644
--- a/include/configs/octeontx3_common.h
+++ b/include/configs/octeontx3_common.h
@@ -74,5 +74,5 @@
 #define CONFIG_SUPPORT_EMMC_BOOT
 #define CONFIG_SUPPORT_EMMC_RPMB
 #define CONFIG_CMD_BKOPS_ENABLE
-#define MMC_SUPPORTS_TUNING
+
 #endif /* __OCTEONTX3_106XX_H__ */
-- 
2.29.0

